src/capi.cpp

```diff
@@ -1276,6 +1292,15 @@ int rtcSetAACPacketizationHandler(int tr, const rtcPacketizationHandlerInit *ini
 	});
 }
 
+int rtcSetRtcpReceivingSession(int tr) {
```

Comment:
If you change to `chainMediaHandler`, this function should be renamed `rtcChainRtcpReceivingSession`.

Reply:
```suggestion
int rtcChainRtcpReceivingSession(int tr) {
```